src/compiler/factory.ts

```diff
@@ -324,17 +324,19 @@ namespace ts {
         const node = <ArrayLiteralExpression>createNode(SyntaxKind.ArrayLiteralExpression, location);
         node.elements = parenthesizeListElements(createNodeArray(elements));
         if (multiLine) {

```

Comment:
`node.multiLine = !!multiLine`? Or does it really need to not exist when not set?